build.gradle.kts

```diff
@@ -127,4 +127,9 @@ tasks {
         into.setFrom(files("ZapVersions-dev.xml", "ZapVersions-2.8.xml", "ZapVersions-2.9.xml"))
         checksumAlgorithm.set("SHA-256")
     }
+
+    register<ZapTask>("generateAddonsYAML") {
```

Comment:
Hi @thc202 I'm not sure about in which task to execute this? or should this be a manual CLI run?

Reply:
A new task is fine, I'd suggest naming it something like `generateWebsiteAddonsData` though.